src/main/java/com/mcmoddev/lib/API/IItemApi.java

```diff
@@ -0,0 +1,21 @@
+package com.mcmoddev.lib.API;
+
+import com.mcmoddev.lib.material.MMDMaterial;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+
+public interface IItemApi extends IMMDApi {
+	default String getApiName() {
+		return "MMD Item->Material registration API";
+	}
+	
+	Item createItem( String typeName, MMDMaterial material );
+	ItemStack createItem( String typeName, MMDMaterial material, int metadata );
+	ItemStack getItem( String typeName, MMDMaterial material );
+	ItemStack getItem( String typeName, MMDMaterial material, int metadata );

```

Comment:
IItemApi seems to imply the creation of a new Item instance with `createItem`. I think it might not be out of the question that "creating" an item does not necessarily create a new Item, but allows variants of the same item encoded in metadata or NBT.

Reply:
hrm... true - perhaps remove the "create" and stick with "get" - and possibly extend it to allow passing in serialized NBT and/or JSON. I'll work on that.